operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -469,7 +470,7 @@ def getServerClusterPortPropertyValue(self, server, clusterListenPortProperty):
       sslListenPort = ssl.getListenPort()
     sslListenPortEnabled = None
     if ssl is not None:
-      sslListenPortEnabled = ssl.isListenPortEnabled()
+      sslListenPortEnabled = ssl.isEnabled()
```

Comment:
The only reference to `getServerClusterPortPropertyValue` is from `validateClusterServersListenPortProperty`. But I cannot find `validateClusterServersListenPortProperty` being called from anywhere. If so this change in line 472 would not make any difference. I may be mistaken though.